### PR TITLE
Add interview with L’Atelier Animation

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: content/blog/2017-06-14-interview-with-latelier-animation.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+ 
+Our infrastructure consists of around 300 render blades, 150 workstations and
+twenty various servers. With the exception of a couple of Macs, everything runs
+on Linux ([CentOS](https://www.centos.org/)) and not a single Windows machine.   
+
+ 
+## What was your pre-Prometheus monitoring experience?
+ 
+At first we went with a mix of [Nagios](https://www.nagios.org/),
+[Graphite](https://graphiteapp.org/), [InfluxDB](https://www.influxdata.com).
```
Comment (grobie) [inline content/blog/2017-06-14-interview-with-latelier-animation.md]: `, and InfluxDB`?

File: content/blog/2017-06-14-interview-with-latelier-animation.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+title: Interview with L’Atelier Animation
+created_at: 2017-06-14
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Philippe Panaite
```
Comment (ppana) [inline content/blog/2017-06-14-interview-with-latelier-animation.md]: +*Continuing our series of interviews with users of Prometheus, Philippe Panaite and Barthelemy Stevens

File: content/blog/2017-06-14-interview-with-latelier-animation.md (hunk 1 above)
Comment (brian-brazil) [inline content/blog/2017-06-14-interview-with-latelier-animation.md]: Done

File: content/blog/2017-06-14-interview-with-latelier-animation.md (hunk 2 above)
Comment (brian-brazil) [inline content/blog/2017-06-14-interview-with-latelier-animation.md]: Done

**Pull request merged**